imperial_coldfront_plugin/admin.py

```diff
@@ -2,7 +2,16 @@
 
 from django.contrib import admin
 
-from .models import GroupMembership
+from .models import GroupMembership, ResearchGroup
+
+
+class ResearchGroupAdmin(admin.ModelAdmin):
+    """Admin configuration for the ResearchGroup model."""
+
+    pass
```

Comment:
This should include a `list_display` to show the names of the groups in the admin page

Reply:
Done.